src/content/docs/style-guide/word-choice/pricing-language-guidelines.mdx

```diff
@@ -17,7 +17,7 @@ These are the primary places in our public docs where pricing- and billing-relat
 * We explain billing impacts in the main pricing and billing-related docs. The main docs in this area are:
   * [Usage-based pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/)
   * [Original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing)
-* When a feature requires a pricing edition (Standard, Pro, Enterprise), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, this is an infrequent need.
+* When a feature requires a specific [pricing edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, this is an infrequent need.
```

Comment:
```suggestion
* When a feature requires a specific [pricing edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, you won't see many of these callouts.
```